src/components/common/Members/MembersTitle.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useCallback, useRef } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import Heading from '~shared/Heading';
+import Icon from '~shared/Icon';
+import { Select, Form } from '~shared/Fields';
+
+import { useMobile } from '~hooks';
+
+import styles from './MembersTitle.css';
+import ColonyDomainSelector from '~common/ColonyHome/ColonyDomainSelector';
+
+const displayName = 'common.Members.MembersTitle';
+
+const MSG = defineMessages({
+  title: {
+    id: `${displayName}.title`,
+    defaultMessage: `{isMobile, select,
+      true {Members}
+      other {Members: }
+    }
+  `,
+  },
+  search: {
+    id: `${displayName}.search`,
+    defaultMessage: 'Search',
+  },
+  searchPlaceholder: {
+    id: `${displayName}.searchPlaceholder`,
+    defaultMessage: 'Search...',
+  },
+  labelFilter: {
+    id: `${displayName}.labelFilter`,
+    defaultMessage: 'Filter',
+  },
+});
+
+interface Props {
+  currentDomainId: number;
+  domainSelectOptions: {
+    value: string;
+    label: string;
+  }[];
+  handleDomainChange: (domainId: number) => void;
+  searchValue: string;
+  handleSearch: (e: any) => void;
+}
+
+const MembersTitle = ({
+  currentDomainId,
+  handleDomainChange,
+  domainSelectOptions,
+  searchValue,
+  handleSearch,
+}: Props) => {
+  const { formatMessage } = useIntl();
+  const searchInput = useRef<HTMLInputElement>(null);
+  const handleFocusRef = useCallback(() => {
+    searchInput?.current?.focus();
```

Comment:
These could all be defined outside the component and take the `searchInput` as an arg

Reply:
Not sure what you mean. Do you want to move searchInput, handleFocusRef, handleMouseEnterRef and handleMouseLeaveRef into `Members.tsx`? Doesnt searchInput still need to be passed as a ref prop into the <input/> in MembersTitle?